config/config.go

```diff
@@ -226,23 +267,11 @@ func (cfg *APIConfig) load() error {
 	}
 	cfg.privateKey = pkey
 	buf.Reset()
-	fd, err = os.Open(cfg.IdentityPath)
-	if err != nil {
-		return fmt.Errorf("failed to open identity file: %w", err)
-	}
-	r = io.LimitReader(fd, MaxFileSize)
-	_, err = buf.ReadFrom(r)
-	_ = fd.Close()
-	if err != nil {
-		return fmt.Errorf("failed to read identity file: %w", err)
-	}
-	err = json.Unmarshal(buf.Bytes(), &cfg.identity)
-	if err != nil {
-		return fmt.Errorf("failed to deserialize device identity: %w", err)
-	}
 	return nil
 }
 
+const magicTenantToken = "<INSERT TENANT TOKEN HERE>"
```

Comment:
@alfrunes The current string used in the edge module seems to be:

```suggestion
const magicTenantToken = "REPLACE_THIS_WITH_YOUR_TOKEN"
```